core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -94,6 +95,407 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     super.tearDown()
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testDescribeUserScramCredentials(quorum: String): Unit = {
+    client = createAdminClient
+
+    // add a new user
+    val targetUserName: String = "tom"
+    client.alterUserScramCredentials(util.Arrays.asList(
+      new UserScramCredentialUpsertion(targetUserName, new ScramCredentialInfo(ScramMechanism.SCRAM_SHA_256, 4096), "123456")
+    )).all.get
+    org.apache.kafka.test.TestUtils.waitForCondition(new TestCondition() {
```

Comment:
this is in scala world, so please use `TestUtils.waitUntilTrue()` instead. For instance:
```scala
    TestUtils.waitUntilTrue(() => client.describeUserScramCredentials().all().get().size() == 1,
      "add one user scram credential timeout")
```